pkg/api/topology.go

```diff
@@ -3,10 +3,62 @@ package api
 import (
 	"net/http"
 
+	"github.com/flanksource/canary-checker/pkg/db"
 	"github.com/flanksource/canary-checker/pkg/topology"
+	"github.com/flanksource/duty/models"
 	"github.com/labstack/echo/v4"
 )
 
+type Tag struct {
+	Key string `json:"key"`
+	Val string `json:"val"`
+}
+
+type TopologyRes struct {
```

Comment:
Use `TopologyResponse`, do not use shortened names for structs unless it is required